common/thorhelper/thorsoapcall.cpp

```diff
@@ -170,6 +170,23 @@ class Url : public CInterface, implements IInterface
         return;
     }
 
+    void cleanPath()
```

Comment:
It looks like cleanPath() is trying to walk the whole string and remove any duplicated '/' 's?
e.g. convert : "POST //WsPackageProcess/myxpath//sub HTTP/1.1"
to "POST /WsPackageProcess/myxpath/sub HTTP/1.1

Is that ok/right?

Could do with brief comment here.

Reply:
Yes all duplicate slashes should be removed, at the beginning, in the middle or at the end of the path...